opendbc/car/gm/carstate.py

```diff
@@ -135,7 +139,9 @@ def update(self, pt_cp, cam_cp, _, __, loopback_cp) -> structs.CarState:
         *create_button_events(self.cruise_buttons, prev_cruise_buttons, BUTTONS_DICT,
                               unpressed_btn=CruiseButtons.UNPRESS),
         *create_button_events(self.distance_button, prev_distance_button,
-                              {1: ButtonType.gapAdjustCruise})
+                              {1: ButtonType.gapAdjustCruise}),
+        *create_button_events(self.lkas_button, prev_lkas_button,
+                              {1: ButtonType.altButton1})
```

Comment:
We already use altButton2 for LKAS on the Honda, please rename that one and use it.